bindings/c/foundationdb/fdb_c.h

```diff
@@ -120,8 +120,13 @@ extern "C" {
     fdb_future_get_error( FDBFuture* f );
 #endif
 
+#if FDB_API_VERSION <= 610
```

Comment:
It looks like the convention is to do the following:

```suggestion
#if FDB_API_VERSION < 620
```